### PR TITLE
Ruby on Rails with unicorn and nginx on Ubuntu 14.04

### PR DESCRIPTION
**Pull request by vaibhavnoob**
A guide on how to deploy rails application using Unicorn and Nginx on Ubuntu 14.04.

### CODE REVIEW

File: docs/websites/ror/ror-with-unicorn-and-nginx-on-ubuntu-14.04.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+##Install Rails
+
+	sudo gem install rails
+
+##Create Rails application
+
+1.	Create a new rails project. We will be using `example` as our project name:
+
+		rails new example
```
Comment (alexfornuto) [inline docs/websites/ror/ror-with-unicorn-and-nginx-on-ubuntu-14.04.md]: At this point a reader following the guide will be in `~/ruby-2.2.3.tar.gz`. Is this where they should be creating their new application?

**Comment by alexfornuto** [comment]
For guides at this level you don't need to specify `nano` or any other text editor. Just tell them what to edit, and trust that the reader knows how to open text files. Otherwise they aren't ready to ready for this guide.

File: docs/websites/ror/ror-with-unicorn-and-nginx-on-ubuntu-14.04.md (hunk 1 above)
Comment (alexfornuto) [inline docs/websites/ror/ror-with-unicorn-and-nginx-on-ubuntu-14.04.md]: Based on L146 I think not.

File: docs/websites/ror/ror-with-unicorn-and-nginx-on-ubuntu-14.04.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+	sudo nano /etc/nginx/nginx.conf
+
+{: .file-excerpt}
+/etc/nginx/nginx.conf
+:	~~~ upstream rails {
+	    # Path to Unicorn socket file
+	    server unix:/home/username/example/shared/sockets/unicorn.sock fail_timeout=0;
+		}
+	~~~
+
```
Comment (alexfornuto) [inline docs/websites/ror/ror-with-unicorn-and-nginx-on-ubuntu-14.04.md]: Tell the reader whenever they need to replace text in a file or command.

**Comment by alexfornuto** [comment]
In addition to the notes on this PR, I've created https://github.com/vaibhavnoob/docs/pull/4 for you and left some additional instruction.

**Comment by vaibhavnoob** [comment]
@alexfornuto: I can't marge your PR due to merge conflict so in another commit I fixed indentation.

**Comment by alexfornuto** [comment]
I suppose I should have specified that you should merge a PR before making additional changes to avoid conflicts. Oh well, live and learn ;)

**Comment by vaibhavnoob** [comment]
By mistake I committed first and forgot to merge PR which resulted in merge conflict :( and I have updated the guide also.

**Comment by alexfornuto** [comment]
Please go back to https://github.com/vaibhavnoob/docs/pull/4 and merge it now. I originally fixed the formatting for the first section of the guide in the hope that you would take the example and complete the rest of the guide. The PR now has indentation fixes for the entire guide.

File: docs/websites/ror/ror-with-unicorn-and-nginx-on-ubuntu-14-04.md [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+
+6. Getting ready for the installation:
+
+	make
+
+7. Issue the following command and it will install Ruby:
+
+	sudo make install
+
+##Install Rails
```
Comment (alexfornuto) [inline docs/websites/ror/ror-with-unicorn-and-nginx-on-ubuntu-14-04.md]: Clearly there shouldn't be an entire section just for one command. How about merging with the previous section?

File: docs/websites/ror/ror-with-unicorn-and-nginx-on-ubuntu-14-04.md [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+
+1. Create a new rails project. We will be using `example` as our project name:
+
+	rails new example
+
+2. Move to the project directory:
+
+	cd example
+
+##Install Unicorn
```
Comment (alexfornuto) [inline docs/websites/ror/ror-with-unicorn-and-nginx-on-ubuntu-14-04.md]: Same as L71

File: docs/websites/ror/ror-with-unicorn-and-nginx-on-ubuntu-14-04.md [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+
+Now create directories which we mentioned in the Unicorn config file:
+
+	mkdir -p shared/pids shared/sockets shared/log
+
+{: .note}
+>
+>Please note that we are still in the Rails application directory
+
+##Install Nginx
```
Comment (alexfornuto) [inline docs/websites/ror/ror-with-unicorn-and-nginx-on-ubuntu-14-04.md]: And again.

**Comment by vaibhavnoob** [comment]
Sorry for the delay. I have done that please check it again.

**Comment by vaibhavnoob** [comment]
any update??

**Comment by alexfornuto** [comment]
@pbzona has made a PR with additional fixes and updates, waiting on you to merge. You can find it here: https://github.com/vaibhavnoob/docs/pull/6

**Comment by vaibhavnoob** [comment]
merged the PR

**Comment by alexfornuto** [comment]
Great, now we just need @kcraig34 to perform a copy edit, and then we'll be all set to publish. Would you prefer PayPal or Linode credit, and what email should we use?

**Comment by vaibhavnoob** [comment]
I would prefer Paypal and where I can give you my email?

**Comment by vaibhavnoob** [comment]
Also when I can expect this to be published on Linode Docs?

**Comment by alexfornuto** [comment]
You can email contribute@linode.com with your address. It will be published as soon as we're done making changes. I see you've merged @kcraig34's copy edits. I see a few more modifications to make, but we can do that internally. Not much longer now...

**Comment by alexfornuto** [comment]
@vaibhavnoob Can you tell me why this guide installs ruby from source instead of using package management? Doing it this way, the user is now responsible for keeping the software up-to-date/

**Comment by vaibhavnoob** [comment]
Well you know package managers provide older version of Ruby and if you need to install a particular version then you have to use third-party tools like RVM and I find it little bit difficult to use so I simply opted for building from source.

**Comment by alexfornuto** [comment]
Congrats! Your guide has been accepted, and will be live shortly. You should receive payment within 24-48 hours. Thanks for contributing to the Linode Guides & Tutorials!

**Pull request merged**